frontend/src/bundles/resume-preview/components/resume-preview/resume-review.tsx

```diff
@@ -0,0 +1,79 @@
+import data from '../../data/resume-preview.json';
+import {
+    Contacts,
+    Design,
+    Education,
+    Experience,
+    Languages,
+    Portfolio,
+    Profile,
+    Recommendations,
+    Socials,
+    Summary,
+    TechStack,
+} from '../components';
+import styles from './styles.module.scss';
+
+const ResumePreview: React.FC = () => {
+    return (
+        <div className={styles.resume_preview__wrapper}>
+            <Profile />
+            <Contacts />
+            <div className={styles.resume_preview__content}>
+                <div className={styles.resume_preview__content_main_section}>
+                    <Summary
+                        data={data.summary}
+                        json_styles={{
```

Comment:
I think we should come up with the different approach to pass json_styles to component, current is hard to read.
Maybe create a separate function that will map json_styles.